lib/SensorFusion/src/LinearKalmanFilter.cpp

```diff
@@ -83,7 +104,17 @@ IKalmanFilter::PositionData LinearKalmanFilter::updateStep(KalmanParameter& kalm
 /******************************************************************************
  * Private Methods
  *****************************************************************************/
+void LinearKalmanFilter::generateMeasurementVector(KalmanParameter& kalmanParameter)

```

Comment:
No doxygen documentation?

Reply:
The Doxygen Documentation is only in the header-File.